tests/project/test_config.py

```diff
@@ -30,33 +28,6 @@ def test_na_minimal_project(project_definition_files: List[Path]):
     assert project["native_app"]["name"] == "minimal"
     assert project["native_app"]["artifacts"] == ["setup.sql", "README.md"]
 
-    from os import getenv as original_getenv
```

Comment:
@sfc-gh-pjob  This section is removed because `generate_local_override_yml` method is not invoked anywhere in the code. Therefore it can be removed from tests.